content/battery.md

```diff
@@ -19,167 +19,155 @@ section: hardware
 
-TLP will take effect upon restart. To see current configuration settings, run this command:
+The settings for the graphics modes on laptops that have switchable Nvidia graphics are outlined in detail [on their own support document](https://support.system76.com/articles/graphics-switch-pop), but to summarize:
 
-```bash
-sudo tlp-stat
-```
+* Integrated Intel graphics mode will provide increased battery life at the cost of 3D performance, and some laptops will not support external displays in that mode.
+* Hybrid graphics allow you to choose to use the more powerful Nvidia GPU when needed, but applications will default to using the lower-power Intel graphics.
+* Nvidia graphics mode runs everything on the Nvidia GPU for maximum perormance, and will use the most battery.
+* Compute mode reserves the Nvidia GPU for tasks that don't output graphics to the screen, like machine learning or data crunching processes.
 
-The program is highly configurable by editing the settings file. Run this command to edit the file:
+Switching graphics modes does require restarting the laptop, which will use up some battery life in the process, so it's recommended that the graphics mode be switched before disconnecting from the AC adapter.
 
-```bash
-sudo gedit /etc/default/tlp
-```
+## Disabling unused wireless devices
 
-Starting with <u>TLP</u> 1.3 the default configuration file has changed so the command for that version is below:
+WiFi and Bluetooth are wireless technologies that use up a small amount of power even when they're not being used, as they monitor for wireless network activity. If you're not using them, disabling either WiFi or Bluetooth can improve battery life, and the "Airplane Mode" setting will disable both for maximum savings. They can be disabled individually via the Wi-Fi and Bluetooth panels in the Settings application by toggling the switch at the top of the window, and Airplane Mode is available underneath it on the Wi-Fi settings panel:
 
-```bash
-sudo gedit /etc/tlp.conf
-```
-
-The `/etc/tlp.conf` file is for user configuration while `/etc/tlp.d/*.conf` file is for drop-in customization snippets.
-
-All of the info about the program can be found with these 2 commands:
-
-```bash
-man tlp
-man tlp-stat
-```
+![disable-wifi](/images/battery/disable-wifi.png)
 
----
+## Monitoring system performance
 
-### Powertop
+Pop!\_OS has a number of different options for monitoring the CPU and GPU use of processes and applications. The more CPU and/or GPU a program is using, the more battery life it will consume as well. If a system's battery isn't lasting as long as expected, it can be useful to see what the system is doing to go through the charge more quickly.
 
-Powertop monitors running processes and applications to determine how much power is being consumed. Powertop can also use application battery usage data to create custom power profiles. We recommend installing and running TLP before using Powertop.
+### `powertop`
```

Comment:
I don't have anything against putting a tool in backtics in a title, but I don't think we do this anywhere else and it's a little difficult to recognize it as a section header. Maybe an additional title word would help, like "The `powertop` tool".